src/core/thread/router_table.cpp

```diff
@@ -368,17 +368,17 @@ Router *RouterTable::GetNeighbor(uint16_t aRloc16)
     return router;
 }
 
-Router *RouterTable::GetNeighbor(const Mac::ExtAddress &aExtAddress)
+Router *RouterTable::FindNeighbor(const Mac::ExtAddress &aExtAddress)
 {
     return FindRouter(Router::AddressMatcher(aExtAddress, Router::kInStateValid));
 }
 
-Router *RouterTable::GetNeighbor(const Mac::Address &aMacAddress)
+Router *RouterTable::FindNeighbor(const Mac::Address &aMacAddress)
 {
     return FindRouter(Router::AddressMatcher(aMacAddress, Router::kInStateValid));
 }
 
-const Router *RouterTable::GetRouter(uint8_t aRouterId) const
+const Router *RouterTable::FindRouterForId(uint8_t aRouterId) const
```

Comment:
nit: `FindRouterById` seems more natural to me. Thoughts?

Reply:
SGTM. I actually used `FindRouterById()` in the commit message (when describing the change) but in the code used the `FindRouterForId()`. 😄

Reply:
Updated in new push.